google/cloud/firestore_v1/aggregation.py

```diff
@@ -54,6 +58,7 @@ def get(
             retries.Retry, None, gapic_v1.method._MethodDefault
         ] = gapic_v1.method.DEFAULT,
         timeout: float | None = None,
+        explain_options: Optional[ExplainOptions] = None,
```

Comment:
Also, probably better to follow existing style (from the timeout argument) and use `ExplainOptions | None` instead of `Optional`

Reply:
> Can you add a * above `explain_options

Thanks for pointing this out, it's great to know! Added.

Reply:
> Also, probably better to follow existing style (from the timeout argument) and use ExplainOptions | None instead of Optional

I'm not quite sure about this, because the type hint style is a mix right now. Maybe we can save it for a docstring overhaul later, when we are certain about which style we will go for?

Reply:
Yeah that works for me!